client/src/ui/molecules/tools-menu/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { OBSERVATORY_TITLE } from "../../../../../libs/constants";
+import { useLocale } from "../../../hooks";
+import { Menu } from "../menu";
+
+import "./index.scss";
+
+export const ToolsMenu = ({ visibleSubMenuId, toggleMenu }) => {
+  const locale = useLocale();
+
+  const menu = {
+    id: "tools",
+    label: (
+      <>
+        Tools <sup className="new">New</sup>
+      </>
+    ),
+    items: [
+      {
+        description: "Write, test and share your code",
+        hasIcon: true,
+        iconClasses: "submenu-icon",
+        label: "Playground",
+        url: `/${locale}/play`,
+      },
+      {
+        description: "Scan a website for free",
+        hasIcon: true,
+        iconClasses: "submenu-icon",
+        label: OBSERVATORY_TITLE,
+        url: `/${locale}/observatory`,
```

Comment:
We shouldn't serve English content on `/fr/`, and we don't take the locale into consideration when linking to headers.

Reply:
Fixed in 20a567dacc.